### PR TITLE
[5255] Add & remove criteria from product page template

### LINKED ISSUE #5255: Add new criteria and remove old criteria - template & styling
### Add new criteria and remove old criteria

- [x] Add all the criteria to match the [spreadsheet](https://docs.google.com/spreadsheets/d/1cOOzLhkoICFIPTUmxQnrH10HZz1qq5eHF51qgj1p8QM/edit#gid=0)
- [x] Remove criteria that are no longer needed 
- [x] Update the styling to match the [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Like on production, if there is no help text then that section should be hidden
- [x] If the answer to the first AI questions is `No` then the other two questions should be hidden

All styling should use styles that are already on production.

Ding will be added in  - Add "privacy" ding #5308 
Criteria can be ordered - Reorder the criteria in the product template #5253

### Desktop comp

What is required to signup?
image

What data does it collect?
image

How does it use this data?
image

How can you control your data?
image

What is the company's known track record of protecting user's data?
image

Can this product be used offline?
image

Artificial Intelligence section
image

---

**🗒  Design -> Dev Handoff Checklist**
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to original design ticket #5152
- [X] Link to related ticket #5234
- [x] Link to [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included - n/a
- [x] Hover state for elements are designed - use existing styles
- [x] Other states for elements are designed (e.g., error state for form) - n/a

### PR DESCRIPTION
**Pull request by KalobTaulien**
Partial fix for #5255 
Related PRs/issues #5255

Related PR: #5280 — I was working off of this PR to reflect the backend changes.

**Notes**:
* I did not add any styling or images. There are TODO comments in here for the next tickets to look at. 
* I copied pre-existing styles (ie `h2.h3-heading`) to keep things similar to how they were before to make the "cleanup" ticket easier to work with. 
* [This is the design](https://app.abstract.com/projects/fa246750-1c19-11e8-b2bf-9db5acc6ed5d/branches/master/commits/4b6da88b41a28a4c1cbbee0f502fa17fe960e90d/files/DBF82218-05B7-47F9-AD7C-8ABDFD109060/layers/BB7D4F2B-6519-45AE-B80F-A70F4F3ECCE9?mode=design&selected=root-BA5F9C19-8B69-4496-9CD1-8E115DD916BC) page I was going off of.

### CODE REVIEW

**Review by mmmavis — COMMENTED** [review]
Hey @KalobTaulien , mind providing a review link? I visited a random product page https://foundation-s-5255-add-r-axgveh.herokuapp.com/en/privacynotincluded/products/percy-cypress/ but it didn't seem to include todos mentioned in the ticket

image

**Comment by KalobTaulien** [comment]
Hey @mmmavis the TODOs are in django template comments. Should be able to see them in the template source

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -89,39 +94,83 @@ <h2 class="h3-heading">{% trans "What is required to sign up?" %}</h2>
...
-          {% include "fragments/signup_requirement.html" with title=third_party_account   value=product.signup_requires_third_party_account    %}
+          {% if product.signup_requires_email %}
+            {% include "fragments/signup_requirement.html" with title=email                 value=product.signup_with_email          %}
+          {% endif %}
+          {% if product.signup_requires_phone %}
+            {% include "fragments/signup_requirement.html" with title=phone                 value=product.signup_with_phone          %}
+          {% endif %}
+          {% if product.signup_requires_third_party_account %}
+            {% include "fragments/signup_requirement.html" with title=third_party_account   value=product.signup_with_third_party    %}
+          {% endif %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:105]: These are not boolean values.
```suggestion
          {% include "fragments/signup_requirement.html" with title=email                 value=product.signup_with_email          %}
          {% include "fragments/signup_requirement.html" with title=phone                 value=product.signup_with_phone          %}
          {% include "fragments/signup_requirement.html" with title=third_party_account   value=product.signup_with_third_party    %}
```

**Review by Pomax — CHANGES_REQUESTED** [review]

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 2]
```diff
@@ -179,6 +227,9 @@ <h2 class="h3-heading">{% trans "Minimum Security Standards" %}</h2>
             {% trans "Manages vulnerabilities" as manages_vulnerabilities %}
             {% trans "Privacy policy"          as privacy_policy %}
 
+            <h2 class="h3-heading">Does this product meet our <u>Minimum Security Standards?</u></h2>
+            {% if product.meets_minimum_security_standards %}Yes{% else %}No{% endif %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:231]:
```suggestion
            {% if product.meets_minimum_security_standards %}
              {% trans "Yes" %}
            {% else %}
              {% trans "No" %}
            {% endif %}
```

**Review by Pomax — CHANGES_REQUESTED** [review]

**Review by Pomax — APPROVED** [review]

**Review by mmmavis — APPROVED** [review]

**Comment by kristinashu** [comment]
Is there a new ticket for the work that didn't get done in this ticket #5255? I don't want to loose things like "If the answer to the first AI questions is `No` then the other two questions should be hidden"

**Comment by Pomax** [comment]
updated the original post so that merging won't close that issue, and updated the issue so that the "field work" and the "styling work" are separate checkboxes.

**Pull request merged**